File: Actividad 1/Actividad 1.3/actividdad a3.cpp
#include <iostream>
using namespace std;

class Estudiantes {
    public:
    string nombre;
    int edad;
    int grado;
    
    //funcion constructor
    Estudiantes(string nameEstudiante, int edadEstudiante, int gradoEstudiante){
        nombre=nameEstudiante;
        edad=edadEstudiante;
        grado=gradoEstudiante;
    }
    
    void tomarExamen(){
        cout<<"Tomando Evaluación..."<<endl;
    }
    void calificarExamen(){
        cout<<"Examen Calificado :)"<<endl;
    }
};

int main(){
    Estudiantes escolar1("Mauricio", 15, 4);
    Estudiantes escolar2("Oscar", 20, 2);
    escolar1.tomarExamen();
    escolar2.calificarExamen();
    return 0;
}

File: Actividad 1/Actividad 1.3/actividdad a3.py
class Estudiantes:
    def __init__(self, nameEstudiante, edadEstudiante, gradoEstudiante):
        self.nombre = nameEstudiante
        self.edad = edadEstudiante
        self.grado = gradoEstudiante

    def tomarExamen(self):
        print("Tomando Evaluación...")

    def calificarExamen(self):
        print("Examen Calificado :)")

escolar1 = Estudiantes("Mauricio", 15, 4)
escolar2 = Estudiantes("Oscar", 20, 2)
escolar1.tomarExamen()
escolar2.calificarExamen()

